dev-site-config/site.config.js

```diff
@@ -1,9 +1,9 @@
-const path = require('path');
+const glob = require('glob');
 const navConfig = require('./navigation.config');
 
-const terraFrameworkPath = path.join(process.cwd(), 'node_modules', 'terra-framework');
-const terraClinicalPath = path.join(process.cwd(), 'node_modules', 'terra-clinical');
-const terraCorePath = path.join(process.cwd(), 'node_modules', 'terra-core');
+const patterns = glob.sync('node_modules/terra-*/lib/terra-dev-site').map(file => (
```

Comment:
@mjhenkes I actually set it up that way initially. The change we need in terra-dev-site is converting our glob pattern to regex. They don't quite match up:

https://github.com/cerner/terra-dev-site/blob/master/scripts/generate-app-config/generatePagesConfig.js#L170

Reply:
I think the fix is to glob, then regex, then glob again, but i haven't dug in as much as i'd like.

Reply:
We'd need something like this: https://github.com/fitzgen/glob-to-regexp

If we pass a glob pattern with `*` it expects anything, but regex thinks of it as multiple times, so it would need converted to `.*`. I didn't dig too much but there might be a different way we could parse the directory name from the entry point.